backend/monitoring/dashboard.py

```
@@ -1439,13 +1442,18 @@ async def get_available_test_suites():
                     'monitoring': '📊'
                 }
                 
+                # Create display name from suite_name and description
+                display_name = suite['suite_name'].replace('_', ' ').title()
+                if suite['description']:
+                    display_name += f" - {suite['description']}"
+                
                 categorized_suites[category]["services"].append({
-                    "title": suite['display_name'] or suite['suite_name'].replace('_', ' ').title(),
+                    "title": legacy_name,
```

Ruff (0.12.2)

Undefined name `legacy_name`

(F821)